templates/geosafe/analysis/create.html

```diff
@@ -615,6 +615,15 @@ <h4 class="modal-title" id="myModalLabel">{% trans "Upgrade Your Browser" %}</h4
             }).addTo(map);
 
             {# add control to select feature on aggregation layer #}
+
+            function reset_button_popover() {
+                popover_content = "Now choose for specific polygons of this aggregation layer. This is optional, you can skip this.";
```

Comment:
Needs to be localized (put trans tag). Don't forget to generate the PO and push to transifex